installer/cluster.go

```diff
@@ -509,10 +717,87 @@ func (c *BaseCluster) bootstrapTarget(t *TargetServer) error {
 		return err
 	}
 
+	return nil
+}
+
+func (c *BaseCluster) getAppEnv(appName string, t *TargetServer) (map[string]string, error) {
+	c.SendLog(fmt.Sprintf("Getting ENV for %s", appName))
+
+	sess, err := t.SSHClient.NewSession()
+	if err != nil {
+		return nil, err
+	}
+	stdout, err := sess.StdoutPipe()
+	if err != nil {
+		return nil, err
+	}
+
+	if err := sess.Start(fmt.Sprintf("curl -u ':%s' -H 'Host: controller.%s' localhost/apps/%s/release", c.ControllerKey, c.oldDomain, appName)); err != nil {
```

Comment:
Interesting. I think the reason I didn't use the below approach is due to not having the pin at this point.